VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -987,6 +1032,8 @@ public void Get_GivenNames()
             Assert.Equal("Middle", ((DeathRecord)JSONRecords[0]).GivenNames[1]);
             Assert.Equal("Madelyn", ((DeathRecord)XMLRecords[0]).GivenNames[0]);
             Assert.Equal("Middle", ((DeathRecord)XMLRecords[0]).GivenNames[1]);
+            Assert.Equal("Madelyn", ((DeathRecord)JSONRecords[2]).GivenNames[0]);
+            Assert.Equal(1, ((DeathRecord)JSONRecords[2]).GivenNames.Length);
```

Comment:
Need to address the warnings you see when you run the tests (in this case not using Assert.Equal(...), using Assert.Single(((DeathRecord)JSONRecords[2]).GivenNames)

Reply:
I had trouble with this one.   See lines 562-564 of DeathRecord_Should.cs....